sources/packages/backend/libs/sims-db/src/entities/cas-supplier.model.ts

```diff
@@ -89,7 +110,7 @@ export class CASSupplier extends RecordDataModel {
   @Column({
     name: "supplier_status_updated_on",
     type: "timestamptz",
-    nullable: true,
+    nullable: false,
```

Comment:
This as `nullable: false` should have the property adjusted to not allow nullable.